src/middlewared/middlewared/plugins/security/update.py

```diff
@@ -109,12 +109,19 @@ async def validate_stig(self, current_cred):
                 'prior to enabling General Purpose OS STIG compatibility mode.'
             )
 
-        if not any([user for user in two_factor_users if 'FULL_ADMIN' in user['roles']]):
+        all_local_admins = await self.middleware.call('privilege.local_administrators')
+        excluded_admins = [user["username"] for user in all_local_admins if user['uid'] < 1000]
+
+        if not any([
+            user for user in two_factor_users
+            if user['uid'] >= 1000 and 'FULL_ADMIN' in user['roles']
+        ]):
             raise ValidationError(
                 'system_security_update.enable_gpos_stig',
                 'At least one local user with full admin privileges must be '
                 'configured with a two factor authentication token prior to enabling '
-                'General Purpose OS STIG compatibility mode.'
+                f'General Purpose OS STIG compatibility mode.  '
+                f'EXCLUDED ACCOUNTS: {", ".join(excluded_admins)}.'
```

Comment:
I think this is wrong. We should just have a separate validation error if there are any immutable (e.g. system) accounts with password authentication enabled.

For example: results of:
```
midclt call user.query '[["immutable", "=", true], ["password_disabled", "=", false], ["locked", "=", false], ["unixhash", "!=", "*"]]'
```

Reply:
Once this is in place in `system.security.update` you'll want to add STIG check to `user.update` for combination of password + immutable +  GPOS STIG enabled.